tidb-in-kubernetes/k8s-deployment.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 部署 k8s
+category: how-to
+---
+
+# 软件版本要求
+
+| 软件名称 | 版本 |
+| --- | --- |
+| Docker | docker-ce 18.09.6 |
+| K8s |  v1.12.2 |
+| CentOS | 7.3 或以上 |
```

Comment:
The minimal kernel version is missing.

Reply:
This [document](https://github.com/pingcap/docs-cn/blob/master/dev/how-to/deploy/hardware-recommendations.md) also does not contain the minimal kernel version requirements.

Reply:
As this [blog](https://www.pingcap.com/blog/try-to-fix-two-linux-kernel-bugs-while-testing-tidb-operator-in-k8s/#operation-steps-1) post suggests, CentOS 7.6 kernel 3.10.0-957 or later is required.